src/runtime/account_loader.zig

```diff
@@ -982,13 +930,13 @@ test "loadAccount allocations" {
         }
     }.f;
 
-    try std.testing.checkAllAllocationFailures(allocator, checkFn, .{});
+    try std.testing.checkAllAllocationFailures(std.testing.allocator, checkFn, .{});
```

Comment:
Small suggestion, the `struct { fn f() void {} }.f` stuff looks pretty awkward, it can be made nicer by instead embracing namespacing:
```zig
const helper = struct {
    fn checkFn(allocator: Allocator) !void {...}
};

try std.testing.checkAllAllocationFailures(std.testing.allocator, helper.checkFn, .{});
```
Also opens the door to more easily adding more local functions without needing to refactor a huge amount or add multiple `struct {...}.f` warts.

Reply:
This wasn't part of my change, and I'm not sure I agree this is an improvement. I'd rather avoid declaring pointless namespaces. I would like to just declare the function name in the current scope, and that's it. That being said, I also agree the way to achieve this in zig is pretty ugly.

I don't have a decisive enough preference here, and it's easy enough for me to change it to the way you like.

c2eb6e676e3585793e28f215c9055b06a5f8b9eb

Reply:
what makes it non-zig if the syntax is valid and it works as intended?

Reply:
I mean non-zig in the sense that it doesn't follow the path of least resistance; it comes from a place of being used to do similar things in other languages, and trying to force zig features to do the same (regardless of whether one would argue that it *should* be able to do the same), where in the end it both reads awkwardly, and is awkward to write. It also offers resistance when refactoring, whether you're changing it to use a dedicated namespace, or you're adding a new function in the same way.

It's not an objectively right or wrong thing, it's moreso just that there's a way the creators of the tool intended it to be used, vs how you want to use it. The subpar aesthetics of the latter serve as an indicator of the former.